DEVELOPMENT.md

```diff
@@ -97,3 +97,33 @@ git commit -m "fix: squashed a super gnarly bug"
 yarn run version patch
 yarn run publish
 ```
+
+## Local Development Tips and Tricks
+
+### Environment Variables
+
+This repository supports handful of environment variables to streamline the local development/testing process.
+
+**`REMIX_DEBUG`**
+
+By default, the Remix `rollup` build will strip any `console.debug` calls to avoid cluttering up the console during application usage. These `console.debug` statements can be preserved by setting `REMIX_DEBUG=true` during your local build.
+
+```sh
+REMIX_DEBUG=true yarn watch
+```
+
+**`REMIX_LOCAL_DEV_OUTPUT_DIRECTORY`**
```

Comment:
This is introduced in https://github.com/remix-run/remix/pull/2187 but documented here to avoid conflicts